clients/pkg/promtail/targets/docker/targetmanager_test.go

```diff
@@ -50,7 +50,7 @@ func Test_TargetManager(t *testing.T) {
 		case strings.HasSuffix(path, "/networks"):
 			// Serve networks
 			w.Header().Set("Content-Type", "application/json")
-			err := json.NewEncoder(w).Encode([]types.NetworkResource{})
```

Comment:
Note to reviewers: `types.NetworkResource` is deprecated, but `network.Inspect` is expected to be the same exact type.